torch/onnx/symbolic_opset12.py

```diff
@@ -13,3 +12,18 @@
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
+def broadcast_tensors(g, tensor_list):
+    tensors = sym_help._unpack_list(tensor_list)
+    if (tensors[0].type().dim() != 2 and tensors[0].type().sizes() != tensors[1].type().sizes()):
+        return _unimplemented("broadcast_tensors", "broadcast for tensors is only implemented in case when there are 2 tensors of the same size")
+    return g.op("prim::ListConstruct", *tensors)
```

Comment:
why is this returning a `prim::ListConstruct` instead of an ONNX op? Is this then handled somewhere else?
Also, is this supposed to be doing some broadcasting?

Reply:
It should probably just return tensor_list.
Actually it's not doing broadcasting, since mse loss implementation in PyTorch requires tensors to be broadcastable, this op just check if the inputs are of the same size and then returns them, otherwise it throws an exception. This is only done in order to support mseloss export to ONNX and is not a complete implementation of broadcastable_tensors op

Reply:
mean square distance accepts only 2 tensors.